### DISCUSSION TITLE
powergrid toggle button does not update database

### DISCUSSION THREAD

**Discussion by smmodules** [Power-Components/livewire-powergrid#1524] [Q&A] [answered]
Hey, guys,

I'm racking my brain over a situation. The toggle button does not update data. I'm using MySql. See the images below.

Please... where am I going wrong?

Image 01
![Img-01](https://github.com/Power-Components/livewire-powergrid/assets/84677076/07629ed7-56b8-4151-9c69-26cb52b35bcf)

Image 02
![Img-02](https://github.com/Power-Components/livewire-powergrid/assets/84677076/bd3c1e80-cd93-436d-8ee8-f5cc109b052f)

Image 03
![img-03](https://github.com/Power-Components/livewire-powergrid/assets/84677076/2acaac2d-1d14-4b0a-9b87-dbe5e4a767c7)

Image 04
![img-04](https://github.com/Power-Components/livewire-powergrid/assets/84677076/a66aee7f-8dc7-4eae-9de5-4e92c9e640c1)

Image 05
![img-05](https://github.com/Power-Components/livewire-powergrid/assets/84677076/35d86e1a-588a-46cc-8af6-aa8cbf35d900)

Image 06
![img-06](https://github.com/Power-Components/livewire-powergrid/assets/84677076/083c577d-32a1-4274-a0ef-67b1bfca64a5)

Image 07
![img-07](https://github.com/Power-Components/livewire-powergrid/assets/84677076/81875cb4-815f-42bc-98d4-54a059adc2c8)

**Comment by dansysanalyst** [comment, marked as answer]
Hi @smmodules,

Wild guess here, but is the field `status` in your Model's `$fillable`?

**Reply by smmodules** [reply]
What a shame!  :) In fact the field was not there. Thank you very much.
